metaflow/metadata/heartbeat.py

```diff
@@ -56,7 +56,9 @@ def _ping(self):
 
     def _heartbeat(self):
         if self.hb_url is not None:
-            response = requests.post(url=self.hb_url, data="{}", headers=self.headers)
+            response = requests.post(
+                url=self.hb_url, data="{}", headers=self.headers.copy()
```

Comment:
is this related to the Otel feature, or some other fix that carried over?

Reply:
Oleg mentioned this in the previous [discussion](https://github.com/Netflix/metaflow/pull/1320#discussion_r1155375173), quote here:
> this and other header .copy() calls are a workaround for https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1729 , we can prob remove this once its fixed upstream